### DISCUSSION TITLE
Terminal selection? Problems with VS Code + Git Bash

### DISCUSSION THREAD

**Discussion by tbeason** [quarto-dev/quarto-cli#1398] [Q&A]
I am on Win10 and use VS Code with Git Bash set as my default terminal. I installed Quarto using the official msi installer, I installed the VS Code extension, and I also followed the posted instructions for setting up Jupyter. 

If I open a Powershell terminal, it is able to find `quarto` and render documents as expected. However, Git Bash simply says
```
$ quarto
bash: quarto: command not found
```
or if I do ctrl + shift + k then I get
```
quarto preview hello.qmd --to pdf --no-browser --no-watch-inputs
ERROR: Jupyter kernel 'julia-1.7' not found.
```

I did not do anything special when I installed Python and R, and Git Bash finds those just fine. Quarto is invisible to it. If I run `which quarto` it says it can't find anything by that name but I see that `/c/Users/beasont/AppData/Local/Programs/Quarto/bin` is in fact on my path...

Made this a discussion instead of an issue as I am uncertain where the problem lies. Hoping someone can help.

**Comment by mcanouil** [comment]
On Windows, it is custom to call a software with its extension, i.e. `quarto.cmd`.
You can check the `PATH` to see if Quarto bin directory is in it.

**Reply by tbeason** [reply]
Alright so using `quarto.cmd` instead does indeed work! But why is that needed here when it is not for Julia, Python, R, etc...?

Yes Quarto is on the path.

So my next question then is how I can make it "just work" in VS Code? Would an ok solution be to define an alias such that calls to `quarto` become `quarto.cmd`? (and how would I do that safely?) Or can I tell VS Code to use Powershell for Quarto commands?

**Reply by mcanouil** [reply]
I do not have the latest version of Quarto on my W10 laptop but VSCode and extensions are up to date.

I am using the latest PowerShell 7.2 (not the native one which is 5.1; https://docs.microsoft.com/en-gb/powershell/scripting/install/installing-powershell-on-windows?view=powershell-7.2) and it just works even without the `.cmd` part.

So it seems to me that your issue is your setup not the tools.

To note, I wouldn't use an emulated terminal especially on Windows 😅

**Reply by tbeason** [reply]
Right it works in PowerShell! Just not in Git Bash. Surely there is a way to tell VS Code to use PowerShell only for this project, so I will look into that.
